pyproject.toml

```diff
@@ -92,32 +94,108 @@ docs = [
     "nbsphinx>=0.7,<0.8.7"
 ]
 
+[tool.hatch.build.targets.wheel]
+packages = ["scvelo"]
+
+[tool.coverage.run]
+source = ["scvelo"]
+omit = [
+    "**/test_*.py",
+]
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+xfail_strict = true
+
 [tool.black]
 include = '\.pyi?$'
 exclude = '''
-/(
-    \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | _build
-  | buck-out
-  | build
-  | dist
-)/
+(
+  /(
+      \.eggs
+    | \.git
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+)
 '''
 
-[tool.isort]
-profile = "black"
-use_parentheses = true
-known_num = "networkx,numpy,pandas,scipy,sklearn,statmodels"
-known_plot = "matplotlib,mpl_toolkits,seaborn"
-known_bio = "anndata,scanpy"
-sections = "FUTURE,STDLIB,THIRDPARTY,NUM,PLOT,BIO,FIRSTPARTY,LOCALFOLDER"
-no_lines_before = "LOCALFOLDER"
-balanced_wrapping = true
-length_sort = "0"
-indent = "    "
-float_to_top = true
-order_by_type = false
+[tool.ruff]
+src = ["."]
+line-length = 119
+target-version = "py38"
+select = [
+    "F",  # Errors detected by Pyflakes
+    "E",  # Error detected by Pycodestyle
+    "W",  # Warning detected by Pycodestyle
+    "I",  # isort
```

Comment:
@WeilerP ruff handles isort here

Reply:
Okay, `ruff` does not yet support custom sections and ordering (see https://github.com/charliermarsh/ruff/issues/2419). I removed import sorting with `ruff` for now, and switched back to `isort` in the pre-commit hooks.